03/app.js

```diff
@@ -29,9 +29,12 @@ if(parentElement) {
     btn.textContent = text;
     // dodanie klasy
     btn.classList.add(attr.className);
-    // dodanie atrybutu
-    btn.setAttribute('title', attr.title);
+    // dodanie atrybutów
+    for (let attrName in attr) {
+        btn.setAttribute(attrName, attr[attrName])
```

Comment:
W JS mamy `.className` natomiast w html jest to już `class`.
Dlatego ten zapis nie jest poprawny.
Mógłbyś to zapisać trochę inaczej tj. `btn[attrName] = attr[attrName]` i powinno być ok